src/components/List/List.js

```diff
@@ -1,7 +1,30 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import NavBar from '../NavBar/NavBar';
+import { useHistory } from 'react-router-dom';
 
-const List = () => {
-  return <div>Current List</div>;
+const List = (props) => {
+  const history = useHistory();
+
+  // Removes Token from Local Storage; trying to implement with Global State
+  const clearToken = (props) => {
+    window.localStorage.removeItem('token');
+    props.setToken('');
+    history.push('/');
+  };
```

Comment:
So I get an error when I try to clear the token, unfortunately: `TypeError: props.setToken is not a function`

Truthfully, I don't fully understand **why** this is a problem, but I think you may need to access the props differently because of how you passed them in in `App.js`. 

```suggestion
const List = ({ token, setToken }) => {
  const history = useHistory();

  // Removes Token from Local Storage; trying to implement with Global State
  const clearToken = () => {
    window.localStorage.removeItem('token');
    setToken('');
    history.push('/');
  };
```